.gitignore

```diff
@@ -1,4 +1,5 @@
 .sass-cache
 node_modules/
 .DS_Store
-.publish
\ No newline at end of file
+.publish
+dist/
```

Comment:
I actually don't think we want to ignore dist.  I'd like to check-in versions in the repo for ease of downloading.